api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsSierraLeone.java

```diff
@@ -0,0 +1,533 @@
+package org.openmrs.module.mirebalais.apploader.apps.patientregistration;
```

Comment:
Not sure what the intent was here and why the whole file was changed?  Also, the class name needs to match the file name, I don't think this will compile?  Was this meant to be included in this commit?

Reply:
This was an error - I will push again with file removed